covalent/__init__.py

```diff
@@ -25,6 +25,7 @@
 from ._dispatcher_plugins import local_redispatch as redispatch  # nopycln: import
 from ._dispatcher_plugins import stop_triggers  # nopycln: import
 from ._file_transfer import strategies as fs_strategies  # nopycln: import
+from ._programmatic.commands import covalent_is_running, covalent_start, covalent_stop
```

Comment:
This will break the SDK-only installation of Covalent since it pulls in `covalent_dispatcher`.

Reply:
I've updated the commands, this will no longer break SDK-only installations @cjao .